css.js

```diff
@@ -152,7 +152,8 @@ class CSSTask extends TaskKitTask {
         warnForDuplicates: false,
         features: {
           customProperties: {
-            variables: this.cssVars
+            variables: this.cssVars,
+            strict: false
```

Comment:
what does this do?

Reply:
That's on the task... Basically avoids fallbacks outputting values but the first resolved one:

Imagine this two cases:

```
variables: {
  size: 1
  'h2-size': 1.2
}
```

Now before this `strict` option, a code like this: `line-height: var(--h2-size, var(--size))` would generate two line-heights:

```css
line-height: 1;
line-height: 1.2
```

With strict being false (which looks misleading IMHO) it just generates the first match (which is the intended behaviour. According to their docs:

> Per specifications, all fallbacks should be added since we can't verify if a computed value is valid or not. This option allows you to avoid adding too many fallback values in your CSS.